models/listingSchema.js

```diff
@@ -0,0 +1,39 @@
+import mongoose from "mongoose";
+import Review from "./reviewSchema.js";
+
+const Schema = mongoose.Schema;
+
+const listingSchema = new Schema({
+    title: {
+        type: String,
+        required: true,
+    },
+    description: String,
+    image: {
+       url: String,
+       filename: String,
+    },
+    price: Number,
+    location: String,
+    country: String,
+    reviews: [
+        {
+            type: Schema.Types.ObjectId,
+            ref: "Review",
+        }
+    ],
+    owner: {
+        type: Schema.Types.ObjectId,
+        ref: "User"
+    }
+});
+
+//will delete reviews associated with listings
```

Comment:
missing <space> after the // and before will.. also this is not a relevant comment.. relevant comment would be
`// Deletes the revies associated with listings` no need of will.. Do the same thing for every comment

Reply:
done